rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/VersionServiceV2.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.rest.service.v2;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.ResponseHeader;
+
+import uk.gov.gchq.gaffer.rest.SystemProperty;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+
+import static uk.gov.gchq.gaffer.rest.ServiceConstants.GAFFER_MEDIA_TYPE;
+import static uk.gov.gchq.gaffer.rest.ServiceConstants.GAFFER_MEDIA_TYPE_HEADER;
+import static uk.gov.gchq.gaffer.rest.ServiceConstants.GAFFER_MEDIA_TYPE_HEADER_DESCRIPTION;
+import static uk.gov.gchq.gaffer.rest.ServiceConstants.OK;
+
+@Path("/graph/version")
+@Api(value = "/version")
+public class VersionServiceV2 {
```

Comment:
I did notice that but it would just adding unnecessary complexity as nothing is going to implement them except the one class. Having an interface only makes sense if we are going to have multiple services/classes that implement it which is not the case, its better to keep it simple for maintainability and readability (ideally the existing ones should be refactored too but that's out of scope).

Reply:
I agree about avoiding redundant interfaces. I'd like @GCHQDev404 to take a look in case they have a different opinion.

Reply:
I have no strong opinions on this.